src/app/code/community/Shopgate/Cloudapi/Block/Checkout/Onepage/Success.php

```diff
@@ -73,37 +73,42 @@ public function getJsonOrderData()
     {
         /** @var Mage_Sales_Model_Order $order */
         $order = Mage::getModel('sales/order')->load($this->getOrderId());
-        /** @var Shopgate_Cloudapi_Model_Frontend_Analytics_LogPurchase $logPurchase */
-        $logPurchase = Mage::getModel('shopgate_cloudapi/frontend_analytics_logPurchase');
-        $logPurchase->setType(Shopgate_Cloudapi_Model_Order_Source::SOURCE_WEBCHECKOUT)
-                    ->setId($order->getId())
-                    ->setAffiliation($order->getStoreName())
-                    ->setRevenueGross($this->formatPrice($order->getGrandTotal()))
-                    ->setRevenueNet($this->formatPrice($order->getGrandTotal() - $order->getTaxAmount()))
-                    ->setTax($this->formatPrice($order->getTaxAmount()))
-                    ->setShippingGross($this->formatPrice($order->getShippingAmount()))
-                    ->setShippingNet(
-                        $this->formatPrice($order->getShippingAmount() - $order->getShippingTaxAmount())
-                    )
-                    ->setCurrency($order->getStoreCurrencyCode())
-                    ->setSuccess(true)
-                    ->setBlacklist(false)
-                    ->setTrackers(array());
+        /** @var Shopgate_Cloudapi_Model_Frontend_Checkout_Onepage_Purchase $purchase */
+        $purchase = Mage::getModel('shopgate_cloudapi/frontend_checkout_onepage_purchase');
+        $purchase->setNumber($order->getId());
```

Comment:
This should be the increment_id, according to the example at https://wiki.shopgate.guru/display/DEV/Web+Checkout+and+Web+Register

Reply:
done